sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -73,6 +73,13 @@
 
 public class PHPSensor implements Sensor {
 
+  /**
+   * Describes if an optimized analysis of unchanged by skipping some rules is enabled.
+   * By default, the property is not set (null), leaving SQ/SC to decide whether to enable this behavior.
```

Comment:
I will add something like: `If SQ/SC doesn't set this property it's disabled (false)`